server/test/fast-food-order.spec.js

```diff
@@ -0,0 +1,25 @@
+import supertest from 'supertest';
+import assert from 'assert';
+import app from '../../app';
+
+const request = supertest(app);
+const path = '/api/v1/orders';
+
+describe('Get all Orders', () => {
+    describe('when Admin request for All fast food order', () => {
+        it('should return all fast food orders', (done) => {
+            request.get(path).end((req, res) => {
+                assert.equal(res.status, 200);
```

Comment:
This doesn't assert that the fast-food is returned, please make this test more concrete.